app/uk/gov/hmrc/gform/controllers/AcknowledgementController.scala

```diff
@@ -45,19 +47,26 @@ class AcknowledgementController @Inject() (
   import controllersModule.i18nSupport._
 
   def showAcknowledgement(formId: FormId, formTemplateId4Ga: FormTemplateId, lang: Option[String]) = auth.async(formId) { implicit request => cache =>
-    renderer.renderAcknowledgementSection(formId, cache.formTemplate, cache.retrievals, lang).map(Ok(_))
+    cache.form.status match {
+      case Submitted => renderer.renderAcknowledgementSection(formId, cache.formTemplate, cache.retrievals, lang).map(Ok(_))
+      case _ => Future.successful(BadRequest)
```

Comment:
Should we not send them back to summary page or even declaration, instead of just throwing back request ??